tests/functional/func_template/template__init__.py

```diff
@@ -0,0 +1,148 @@
+# coding=utf-8
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+"""Setup for the test package $TEST_NAME$.
+
+WARNING!!!
+This is a generated test skeleton remove the parts not required by the tests.
+WARNING!!!
+
+"""
+
+from subprocess import CalledProcessError
+
+import multiprocessing
+import os
+import shutil
+import subprocess
+import sys
+import time
+import uuid
+
+from functional import TEST_ENV
+from functional import __REPO_ROOT
+
+from libtest import codechecker
+from libtest import env
+from libtest import get_free_port
+from libtest import project
+
+# Stopping event for CodeChecker server.
+__STOP_SERVER = multiprocessing.Event()
+TEST_WORKSPACE = None
+
+
+def setup_package():
+    """Setup the environment for testing $TEST_NAME$."""
```

Comment:
Yes, I thought about that, but these functional test setup parts are quite specific for each test, and grouping them together will give less flexibility at test configuration and initialization.

If they are moved out into another lib they still needed to be called.
Most of what they do is initialize api clients, read up configuration files from the test workspace.